build/test.go

```diff
@@ -88,7 +83,51 @@ func (d *Driver) testFixtures(image string) error {
 			AutoRemove: true,
 			Binds:      []string{mnt},
 		},
-	})
+	}
+	// run tests
+	if err := d.runContainer(cli, nil, opts); err != nil {
```

Comment:
Maybe I'm missing something here, but why are we running twice? Can we not—when benchmarking is enabled—include the bench flags in the original invocation?

Reply:
In this case, an output of the test will be mixed with benchmark results. I wanted to avoid it.